File: RepoTransBench/target_projects/Python/C++/gregmalcolm_python_koans/public_tests/test_public_helper.cpp
#include "gtest/gtest.h"
#include "lib/helper.h"

TEST(TestPublicHelper, PublicClsNameForBool) {
    EXPECT_EQ(cls_name(true), "bool");
}

TEST(TestPublicHelper, PublicClsNameForTuple) {
    std::tuple<int> t(1);
    EXPECT_EQ(cls_name(t), "tuple");
}

File: RepoTransBench/target_projects/Python/Rust/gregmalcolm_python_koans/public_tests/test_public_helper.rs
use crate::runner::helper::*;

#[test]
fn test_public_cls_name_for_bool() {
    let b = true;
    assert_eq!(cls_name(b), "bool");
}

#[test]
fn test_public_cls_name_for_tuple() {
    let t = (1,);
    assert_eq!(cls_name(t), "tuple");
}